### DISCUSSION TITLE
Validation d'une date en format MM/DD

### DISCUSSION THREAD

**Discussion by ageliinas** [MTESSDev/FRW#152] [Annonces]
Bonjour,
Nous avons dû utilisé un champ texte avec le format MM/DD pour saisir seulement le mois et le jour selon nos demandes. Par contre il a été soulevé qu'il est possible d'inscrire des dates non-existantes tels que 02/31 (31 février) par exemple. Serait-ce possible que cela soit corrigé?

J'ai essayé de mettre une validation personnalisé, mais je n'ai pas réussi à réglé le problème.

Merci,
Bonne journée!

**Comment by j0ber** [comment]
Bonjour,

C'est "by design" dans VueFormulate. 
![image](https://github.com/MTESSDev/FRW/assets/26974817/2ff43d31-85bd-4a4d-ba48-b34941739c00)

Dans un monde idéal quand on travaille avec des dates on utilise un champ de type date et non un champ texte avec une validation date.

Quel est le scénario dans lequel vous avez uniquement besoin du mois et du jour?

Pour régler ce problème il faudra développer une validation personnalisée.

**Reply by ageliinas** [reply]
Bonjour,
Les demandeurs ont trouvés que c'était trop 'confidentiel' de demander la date de naissance complète d'où le pourquoi nous avons dû trouver une alternative pour seulement demander le mois et le jour.

**Comment by j0ber** [comment]
Ok je vois... Et juste comme ça vous faites quoi avec un mois/jour?   (J'essaie de comprendre)

Cela dit pour votre besoin il faudra ouvrir une issue avec votre besoin.

Pour la priorisation demande à Florent de contacter notre PO.

**Reply by ageliinas** [reply]
Selon les informations, c'est pour le besoin d’identifier la personne qui représente un organisme ou un particulier, mais ils ne veulent que le mois et le jour pour être conforme aux règles de PRP tout en respectant la GIC, il faut limiter le nombre de renseignements demandés. Bref, c'est ce que j'ai comme information en tant que front-end.
